components/mediation/mediators/entitlement-mediator/org.wso2.micro.integrator.identity.entitlement.mediator/pom.xml

```diff
@@ -0,0 +1,111 @@
+        <dependency>
+            <groupId>org.apache.ws.commons.axiom.wso2</groupId>
+            <artifactId>axiom</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.synapse</groupId>
+            <artifactId>synapse-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.logging</groupId>
+            <artifactId>pax-logging-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.commons</groupId>
+            <artifactId>org.wso2.carbon.identity.entitlement.stub</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.entitlement.stub</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.authenticator.stub</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>libthrift.wso2</groupId>
+            <artifactId>libthrift</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>slf4j.wso2</groupId>
+            <artifactId>slf4j</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.ei</groupId>
+            <artifactId>org.wso2.micro.integrator.core</artifactId>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.wso2.micro.integrator.identity.entitlement.*,
+                        </Export-Package>
+                        <Import-Package>
+                            !javax.xml.namespace,
+                            org.apache.commons.logging.*; version="${import.package.version.commons.logging}",
+                            !org.osgi.framework,
+                            !javax.servlet,
+                            !javax.servlet.http,
+                            !org.apache.axis2.databinding,
+                            !org.apache.axis2.databinding.utils.reader,
+                            !org.pache.axis2.databinding.utils.writer,
+                            !org.wso2.carbon.identity.entitlement.mediator.*,
+                            org.wso2.carbon.identity.entitlement.proxy.*,
+                            org.wso2.carbon.identity.entitlement.stub.*;version="${carbon.identity.imp.pkg.version}",
+                            org.wso2.carbon.authenticator.stub.*;version="${carbon.kernel.imp.pkg.version}",
```

Comment:
Are we packing org.wso2.carbon.identity packages?

Reply:
yes , these were previously packed in mediation, now moved here.